functions-and-scope.js

```diff
@@ -42,6 +84,32 @@ const grades = [9, 8, 5, 7, 7, 4, 9, 8, 8, 3, 6, 8, 5, 6];
 
 // ---- Verwachte uitkomst: 6.642857142857143
 
+// STAPPENPLAN
+// [X] 1    => schrijf een loop die iedere indexwaarde in de array checkt, ongeacht de lengte (.length gebruiken)
+// [X] 2    => Schrijft een 'actie' die het gemiddelde cijfer van de array (grades) uitrekent (de som van alle cijfers deelt door het totaal aantal cijfers in de array)
+// [X]  2.1 => bepaal de waarde van de som van alle cijfers in de array
+// [X]  2.2 => bepaal het aantal cijfers in de array (array.length)
+// [X]  2.3 => deel de totale waarde van de cijfers door het aantal cijfers in de array
+// [X]  2.4 => sla de uitkomst van 2.3 op
+// [X] 3    => Log het gemiddelde (oftewel: leg de returnwaarde vast in een variabel en log de variabel)
+
+// Declaratie van een overschrijfbare variabele waar de totale waarde van de cijfers in geschreven gaat worden. Ik declareer de beginwaarde (0).
+let output = 0;
+
+// Een loop die alle indexwaardes van de array checkt, ongeacht de lengte
+for (let i = 0; i < grades.length; i++) {
+
+   // Variabel output schrijf ik over zodat hij de voorgaande waarde + de nieuwe waarde is (waarbij de nieuwe waarde het cijfer is van de indexwaarde die op dat moment door de loop gecheckt wordt.)
+   output = output + grades[i];
```

Comment:
Dit kun je versimpelen:
```suggestion
   output += grades[i];
```